lib/howitzer/meta/entry.rb

```diff
@@ -0,0 +1,60 @@
+module Howitzer
+  module Meta
+    # This class provides access to meta information entities
+    class Entry
+      attr_reader :context
+
+      def initialize(context)
+        @context = context
+      end
+
+      # Returns array of elements
+      # @return [Array]
+      def elements
+        @elements ||= context
+                      .private_methods
+                      .grep(/\A(?!wait_)\w+_element\z/)
```

Comment:
There are another methods, like within_xxx_element

Reply:
within_ and has_xxx_element are public methods. Lookup for element methods made within private methods.